src/v2/view-builder/components/AuthenticatorFooter.js

```diff
@@ -3,6 +3,21 @@ import { getSwitchAuthenticatorLink } from '../utils/LinksUtil';
 
 export default BaseFooter.extend({
   links() {
-    return getSwitchAuthenticatorLink(this.options.appState);
+    const switchAuthenticatorLink = getSwitchAuthenticatorLink(this.options.appState);
+
+    const factorPageCustomLink = [];
+    const helpLinksFactorPageLabel = this.options.settings.get('helpLinks.factorPage.text');
+    const helpLinksFactorPageHref = this.options.settings.get('helpLinks.factorPage.href');
+
+    if (helpLinksFactorPageLabel && helpLinksFactorPageHref) {
+      factorPageCustomLink.push({
+        'type': 'link',
+        'label': helpLinksFactorPageLabel,
+        'name' : 'factorPageHelpLink',
+        'href': helpLinksFactorPageHref,
+      });
```

Comment:
nit: As this is just a javascript object, key here do not need the single quotes. it can be written as
```
factorPageCustomLink.push({
        type: 'link',
        label: helpLinksFactorPageLabel,
        name: 'factorPageHelpLink',
       href: helpLinksFactorPageHref,
      });
```
Also, usually there is space just after the `:` not before